core/payment-driver/zksync/src/zksync/wallet.rs

```diff
@@ -135,7 +151,7 @@ pub async fn make_transfer(details: &PaymentDetails, nonce: u32) -> Result<Strin
         .nonce(nonce)
         .str_to(&details.recipient[2..])
         .map_err(GenericError::new)?
-        .token(ZKSYNC_TOKEN_NAME)
+        .token(token.as_ref()) // TODO: use `token`
```

Comment:
Is this `TODO` still relevant? Looks like already using `token`.

Reply:
nope, removed :)